programmers/완주하지-못한-선수.js

```diff
@@ -0,0 +1,15 @@
+function solution(participant, completion) {
+  const marathon = {};
+
+  for (let player of participant) {
+    player in marathon ? (marathon[player] += 1) : (marathon[player] = 1);
+  }
+
+  for (let finishPlayer of completion) {
+    marathon[finishPlayer] -= 1;
+  }
+
+  for (let noCompletion in marathon) {
+    if (marathon[noCompletion] === 1) return noCompletion;
+  }
```

Comment:
```suggestion
  for (let comp of completion) {
    if (players[comp] > 0) players[comp] -= 1;
    if (players[comp] === 0) delete players[comp];
  }
  return Object.keys(players).toString();
```
저는 이렇게 한 번 풀어봤어요 참고해보시고 이해 안되는 부분이 있다면 말씀해주세요